tests/test_client.py

```diff
@@ -27,6 +27,12 @@ async def process(self, docs, **kwargs):
         return results
 
 
+class ErrorExec(Executor):
+    @requests
+    def foo(self, docs, **kwargs):
+        1 / 0
```

Comment:
raise an error here,  do not do like this

Reply:
hahahaha